README.md

```diff
@@ -67,6 +67,11 @@ You may also wish to visit Intel Media Server Studio [support page](https://soft
 * [Tutorials Overview](./doc/tutorials/mediasdk-tutorials-readme.md)
 * [Tutorials Command Line Reference](./doc/tutorials/mediasdk-tutorials-cmd-reference.md)
 
+# Intel Linux Media Stack
+Media SDK is a part of Intel software stack for graphics:
```

Comment:
This belongs to the very first paragraph "# Intel® Media SDK", i.e. move up, please.